pkg/operator/reports.go

```diff
@@ -869,6 +934,47 @@ func (op *Reporting) queueDependentReportsForReport(report *metering.Report) err
 	return nil
 }
 
+// for each report in the namespace, find ones that depend on the report passed into the function
+func (op *Reporting) reportsDependOnThisReport(report *metering.Report, reports []*metering.Report) (bool, string) {
+	var depReportName = ""
+	for _, otherReport := range reports {
+		deps, err := op.getReportDependencies(otherReport)
+		if err != nil {
+			op.logger.Errorf("was unable to get dependencies for report: %s, while looking for dependent reports", report.Name)
+			return true, depReportName
+		}
+		// If a Report has a dependency on the passed in report, we're done
+		for _, dep := range deps.Reports {
+			if dep.Name == report.Name {
+				depReportName = dep.Name
+				return true, depReportName
+			}
+		}
+	}
+	return false, depReportName
+}
+
+// Look for all ReportQueries in the namespace
+func (op *Reporting) reportQueriesDependOnThisReport(report *metering.Report, reportQueries []*metering.ReportQuery) (bool, string) {
+	var depQueryName = ""
+	// for each report in the namespace, find queries that depend on the report passed into the function.
+	for _, reportQuery := range reportQueries {
+		deps, err := op.getQueryDependencies(reportQuery.Namespace, reportQuery.Name, nil)
+		if err != nil {
+			op.logger.Errorf("was unable to get dependencies for report: %s, while looking for dependent queries")
+			return true, ""
+		}
+		// If this ReportQuery has a dependency on the passed in report, we're done
+		for _, dep := range deps.Reports {
+			if dep.Name == report.Name {
+				depQueryName = dep.Name
+				return true, depQueryName
```

Comment:
Same here, no need to initialize the `depQueryName` when we can just return the `dep.Name` outright.